GOVERNANCE.rst

```diff
@@ -384,24 +431,32 @@ If the proposed change cannot gather a 2/3 majority of the votes cast,
 the CC tries to resolve the deadlock.
 
 -  The CC will use consensus seeking.
--  If no consensus can be found within a month, the decision is made
-   through a simple majority vote (with tie breaking) among the CC
+-  If no consensus can be found within twenty working days excluding week-ends
```

Comment:
```suggestion
-  If no consensus can be found within twenty working days excluding weekends
```

Reply:
appears I'm "old-fashioned", might have been reading too many old books.

A poster puts it nicely [here on quora](https://www.quora.com/Which-is-correct-weekend-or-week-end)

"
Weekend is in 2021 (and has been for many decades) the dominant form.

If you were to use week-end in writing, you would probably be labeled as old-fashioned or as fond of outdated language.

Still, there was a time when weekend was inconceivable, a time when it would have been almost obscene.

To write, around the 1940s:

Where have you been for the weekend?

Would've been just like saying, in 2021:

Who will be the garbage taker outer today?

(Which, by the way, you can translate into English by saying: Who will take out the garbage today?)

Probably one day such an expression will become standard English too.

Back to your question, the turning point was somewhere in the fifties, when weekend was introduced and gradually gained popularity over week-end.

Again, before this time week-end was the only form.
"